Gemfile

```diff
@@ -106,13 +106,14 @@ end
 group :test do
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '>= 2.15'
+  gem 'rails-controller-testing'
```

Comment:
add version

Reply:
For what purpose, actually?

Reply:
there may occur some dependency conflicts in future if the version changes